doc/api/deprecations.md

```diff
@@ -2668,6 +2668,35 @@ Type: Documentation-only
 
 Use [`request.destroy()`][] instead of [`request.abort()`][].
 
+<a id="DEP0XX2"></a>
+### DEP0XX2: `repl.inputStream` and `repl.outputStream`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/?????
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+The `repl` module exported the input and output stream twice. Use `.input`
+instead of `.inputStream` and `.output` instead of `.outputStream`.
+
+<a id="DEP0XX1"></a>
+### DEP0XX1: `repl._builtinLibs`
```

Comment:
```suggestion
<a id="DEP0XXX"></a>
### DEP0XXX: `repl._builtinLibs`
```

Reply:
I have to distinguish both to test them, thus I kept this one as is. Our tooling will complain about all of these anyway, since I updated the matching process quite a while ago.